osf/models/preprint_provider.py

```diff
@@ -15,7 +15,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     name = models.CharField(null=False, max_length=128)  # max length on prod: 22
     logo_name = models.CharField(null=True, blank=True, max_length=128)  # max length on prod: 17
     header_text = models.TextField(default='', blank=True)
-    description = models.CharField(null=True, blank=True, max_length=256)  # max length on prod: 56
+    description = models.TextField(null=True, blank=True)
```

Comment:
TextField and CharFields should be `null=False` unless there's a very good reason. `blank=True` is the preferred way and it will keep an empty string in the field if it's not provided.